javascript/javascript1/week4/hw4.js

```diff
@@ -0,0 +1,86 @@
+let todo = [];
+let myName = '';
+const month = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+function getReply(command) {
+
+    if (command.includes('Hello my name is Hamed Hello my name is Hamed')) {
+        return 'Please tell me just once your information';
+
+    } else if (command === 'Hello my name is Hamed') {
+        myName = command.split(' ');
+        return 'nice to meet you ' + myName[4];
```

Comment:
No need to set myName to an array of words (when it's technically _not_ an array of words), you can immediately do `myName = command.split(" ")[4]` (and then also fix line 18).